profiles/netcoreall.props

```diff
@@ -0,0 +1,14 @@
+<Project>
+  <!-- All .NET Core / .NET Standard TFMs -->
+  <PropertyGroup>
+    <FakeItEasyTargetFrameworks>netstandard1.6;netstandard2.0;netstandard2.1</FakeItEasyTargetFrameworks>
+    <ValueTaskExtensionsTargetFrameworks>netstandard1.6;netstandard2.0;netstandard2.1</ValueTaskExtensionsTargetFrameworks>
+    <ExampleTargetFrameworks>netcoreapp3.0</ExampleTargetFrameworks>
+    <SpecsTargetFrameworks>netcoreapp1.0;netcoreapp2.1;netcoreapp3.0</SpecsTargetFrameworks>
+    <IntegrationTestsTargetFrameworks>netcoreapp1.0;netcoreapp2.1;netcoreapp3.0</IntegrationTestsTargetFrameworks>
+    <UnitTestsTargetFrameworks>netcoreapp1.0;netcoreapp2.1;netcoreapp3.0</UnitTestsTargetFrameworks>
+    <AnalyzerTestsTargetFrameworks>netcoreapp2.1;netcoreapp3.0</AnalyzerTestsTargetFrameworks>
+    <TestHelpersTargetFrameworks>netstandard1.6;netstandard2.0;netstandard2.1</TestHelpersTargetFrameworks>

```

Comment:
Why a netstandard2.1 here but not in full?

Reply:
A mistake. TestHelpers doesn't need a netstandard2.1 target

Reply:
And it also shouldn't be netstandard2.1 for the netcore3.0 profile